packages/cloud-cognitive/src/components/PageHeader/PageHeader.test.js

```diff
@@ -725,4 +725,15 @@ describe('PageHeader', () => {
 
     warn.mockRestore(); // Remove mock
   });
+
+  test.only('PageHeader grid settings narrow and fullWidth', () => {
```

Comment:
I'm not sure we want to keep the `.only` here. This will cause `PageHeader.test.js` to [only run this one test](https://jestjs.io/docs/api#testonlyname-fn-timeout), right?

Reply:
Ooops thanks @matthewgallo